bc_obps/service/tests/test_user_operator_service_v2.py

```diff
@@ -139,3 +139,22 @@ def test_list_user_operators_v2():
         )
         assert user_operators_sorted_by_status.first().status == UserOperator.Statuses.APPROVED
         assert user_operators_sorted_by_status.last().status == UserOperator.Statuses.PENDING
+
+    @staticmethod
+    def test_create_operator_and_user_operator_v2():
```

Comment:
I didn't do a full test for this because I didn't want to get bogged down in tech debt; I just tested the bits relevant to the AC of this card